include/controls.h

```diff
@@ -0,0 +1,21 @@
+#ifndef CONTROLS_H
```

Comment:
This could be swapped out for `#pragma once` for consistency with our other code. An include guard like this is great practice though, and will do the same thing.